hooks.lua

```diff
@@ -9,14 +9,47 @@ local b64e = ie.require("mime").b64
 irc.hooks = {}
 irc.registered_hooks = {}
 
+local colour_codes = {
+	[0] = "white",
+	[1] = "black",
```

Comment:
You only really need the index for [0]. The others are auto-numbered starting at 1.